libs/log/handler/colors.go

```diff
@@ -0,0 +1,56 @@
+package handler
+
+import "github.com/fatih/color"
+
+// ttyColors is a slice of colors that can be enabled or disabled.
+// This adds one level of indirection to the colors such that they
+// can be easily be enabled or disabled together, regardless of
+// global settings in the color package.
+type ttyColors []*color.Color
+
+// ttyColor is an enum for the colors in ttyColors.
+type ttyColor int
+
+const (
+	ttyColorInvalid ttyColor = iota
+	ttyColorTime
+	ttyColorMessage
+	ttyColorAttrKey
+	ttyColorAttrSeparator
+	ttyColorAttrValue
+	ttyColorLevelTrace
+	ttyColorLevelDebug
+	ttyColorLevelInfo
+	ttyColorLevelWarn
+	ttyColorLevelError
+
+	// Marker for the last value to know how many colors there are.
+	ttyColorLevelLast
+)
+
+func newColors(enable bool) ttyColors {
+	ttyColors := make(ttyColors, ttyColorLevelLast)
+	ttyColors[ttyColorInvalid] = color.New(color.FgWhite)
```

Comment:
Will this work in white terminal as well?

Reply:
The primary ANSI colors are properly mapped in different themes, I commented below with a couple samples.

Reply:
Just remember we had to get rid of FgWhite and FgBlack as it didn't work on strictly white and black terminals
https://github.com/databricks/cli/pull/435